app/src/main/java/com/asfoundation/wallet/ui/iab/IabActivity.kt

```diff
@@ -44,6 +44,9 @@ class IabActivity : BaseActivity(), IabView, UriNavigator {
 
   @Inject
   lateinit var billingAnalytics: BillingAnalytics
```

Comment:
Since you're cleaning this class i suggest you move

 inAppPurchaseInteractor.savePreSelectedPaymentMethod(
        bundle.getString(PRE_SELECTED_PAYMENT_METHOD_KEY))

into the presenter